pkg/minikube/cruntime/containerd.go

```diff
@@ -79,9 +79,11 @@ func (r *Containerd) Available() error {
 }
 
 // Enable idempotently enables containerd on a host
-func (r *Containerd) Enable() error {
-	if err := disableOthers(r, r.Runner); err != nil {
-		glog.Warningf("disableOthers: %v", err)
+func (r *Containerd) Enable(disableOtherContainerEngines bool) error {
```

Comment:
Prefer shorter variable names for a short scope, such as `disableOthers`. Leave the long variable names for globals.

Reply:
Context: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Hi, although I agree, I can't do it. We have a function called `disableOthers` already.
I can't think about another good short name.
Do you have any suggestions ?